### PR TITLE
styling calculator

### PR DESCRIPTION
**Pull request by Khaled-AlKhateeb**
##Tasks Done ✔️ 

- Created the Calculator Component.
- Added styles to match the UI template.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,10 @@
...
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
```
Comment (Davidosky007) [inline src/App.js]: * Kindly make all components class-based as stated in the [project requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components%20(All%20components%20should%20be%20class%2Dbased).)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Khaled-AlKhateeb 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking readme file 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Davidosky007) [reply src/App.js]: * Also Delete all the boilerplate from CRA (text, images, styles). (ie `logo.svg`)

**Comment by Khaled-AlKhateeb** [comment]
kindly Mr.@Davidosky007, can you please recheck the same branch I pushed ( the components branch) which I created a pull request and asked a code review for it, because I believe I did everything you asking me to do already.

**Comment by Davidosky007** [comment]
> kindly Mr.@Davidosky007, can you please recheck the same branch I pushed ( the components branch) which I created a pull request and asked a code review for it, because I believe I did everything you asking me to do already.

Your components should be in class component but your app.js component is written as a functional component, kindly fix this

**Review by Wes-Isaac — APPROVED** [review]
Hi @Khaled-AlKhateeb,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Use of gitflow 👍.
- Professional README 💯.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**